chombot/src/config.rs

```diff
@@ -0,0 +1,164 @@
+#![allow(clippy::struct_excessive_bools)]
+
+use std::collections::HashMap;
+use std::fs;
+use std::ops::{Deref, DerefMut};
+use std::path::PathBuf;
+
+use async_trait::async_trait;
+use chombot_common::tournaments_watcher::notifier::TournamentWatcherChannelListProvider;
+use poise::serenity_prelude::ChannelId;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Eq, PartialEq, Hash, Serialize, Deserialize)]
+#[serde(transparent)]
+pub struct GuildId(String);
```

Comment:
Why create it from scratch instead of using serenity's [GuildId](https://docs.rs/serenity/0.11.6/serenity/model/id/struct.GuildId.html)? Is it like some TOML's limitation?

Reply:
Unfortunately, yes - TOML seem to only support Strings as the map keys. I'm not to happy about this, but it is what it is.